tests/test_result_to_dict.py

```diff
@@ -35,6 +35,14 @@ class Data(Base):
     data_enum = Column(Enum(MyEnum))
 
 
+def get_object(result):
+    obj = result.get('object')
+    if obj and isinstance(obj, str):
+        dict_value = json.loads(obj)
+        return dict_value
+    raise ValueError("Object values must be a type string")

```

Comment:
not sure we want to raise ValueError here. 
what is the reason?

Reply:
if anything goes wrong we will get to know that, just for test purposes